deploy/olm-catalog/portworx/portworxoperator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -69,71 +69,88 @@ spec:
     mediatype: image/png
   description: |
-    ### Storage Cluster Spec Generator
+    Portworx-Enterprise is the most widely-used and reliable cloud-native
```

Comment:
Has this been vetted by the marketing team?

Reply:
All the contents in the description is from Vick.